### PR TITLE
Create nashik.md

### PR DESCRIPTION
**Pull request by dipaktelangre-training**
All the places that can see in the nashik

### CODE REVIEW

**Comment by dipaktelangre** [comment]
@dipaktelangre-training Can you please add more places

File: places/nashik.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 - Wine yard
 - Ramkund 
+- Sita gupha
```
Comment (dipaktelangre) [inline places/nashik.md]: How far is this ?

Comment (dipaktelangre-training) [inline places/nashik.md]: Do we need to mention it ?

Comment (dipaktelangre) [inline places/nashik.md]: Yep !

**Comment by dipaktelangre** [comment]
This request PR closes #3

**Pull request merged**